### PR TITLE
Fixes #90 update readme about setting up ssh

### LINKED ISSUE #90: Update README with instructions on how to setup SSH keys
### This is a
- [x] Bug report
- [ ] Feature request

### Current behaviour

README does not contain any documentation about the need to setup keys

### Expected behaviour

README should contain instructions on setting up keys

### PR DESCRIPTION
**Pull request by pri22296**
## Description

This PR makes the following changes

- Updates **README** about setting up **ssh**
- Adds description about the **OAUTH_TOKEN** environment variable. refer this [comment](https://github.com/fossasia/yaydoc/pull/95#issuecomment-307550459)
- Removes *creating gh-pages branch* from prerequisites as that is not the case.
- Adds new default values of **PROJECTNAME**, **AUTHOR** and **VERSION** as per PR #84 

## Related Issue

Fixes #90 

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

**Comment by pri22296** [comment]
@imujjwal96 @sch00lb0y Please review

**Comment by imujjwal96** [comment]
I think we can mention our .travis.yml file as a sample for ssh keys

File: README.md [hunk 1]
```diff
@@ -48,4 +46,12 @@ script:
...
 - chmod +x ./generate.sh
 - ./generate.sh
-```
\ No newline at end of file
+```
+
+## Using SSH
+Additionally If you have ssh setup on your repository, yaydoc prioritizes it over Github Personal token. To setup ssh follow these steps. 
+
+- generate a new ssh key following this [Guide](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/#generating-a-new-ssh-key)
```
Comment (imujjwal96) [inline README.md]: Start the statement with a capital letter

Comment (pri22296) [inline README.md]: @imujjwal96 I have made the changes.

**Pull request merged**